skrf/calibration/calibration.py

```diff
@@ -5780,21 +5800,18 @@ def determine_reflect(thru_m, reflect_m, line_m, reflect_approx=None,
     d = -det(thru_m.s)
     f = -thru_m.s[:,1,1]
 
-    gam = (f-d/x)/(1-e/x)
-    b_A = (e-y)/(d-b*f)
+    gam = (f-d/x)/(1-e/x) # equation (40) 
+    b_A = (e-b)/(d-b*f)  # equation (41): beta/alpha
 
     w1 = reflect_m.s[:,0,0]
     w2 = reflect_m.s[:,1,1]
 
-    a = sqrt(((w1-y)*(1+w2*b_A)*(d-y*f))/\
+    # equation (45)
+    a = sqrt(((w1-b)*(1+w2*b_A)*(d-b*f))/\
             ((w2+gam)*(1-w1/x)*(1-e/x)))
-
-    out = []
-    for rootChoice2 in [1,-1] :
-        a= a*rootChoice2
-        unknownReflectS = (w1-y)/(a*(1-w1/x))
-        out.append(unknownReflectS)
-
+        
```

Comment:
There are few of these new lines with spaces. Would be nice to just have newline without the spaces.